kalina-georgieva/Homework-2/Calculator/calculator.css

```diff
@@ -0,0 +1,61 @@
+            body{
```

Comment:
@kalina-g 
The visualization of the code is probably related to git, but just to mention - always format your code : )